pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTReferenceType.java

```diff
@@ -39,4 +39,14 @@ public boolean isArray() {
         return arrayDepth > 0;
     }
 
+
+    /**
+     * Returns the type of this node. If this node represents an
+     * array type, then the returned type is an array type of the
+     * correct dimensions ({@link #getArrayDepth()}).
+     */
+    @Override
+    public Class<?> getType() {
+        return super.getType();
```

Comment:
this override just calls super, it should simply be removed as per [`UselessOverridingMethod`](https://pmd.github.io/pmd-6.1.0/pmd_rules_java_design.html#uselessoverridingmethod).

I don't think the doc is saying anything unexpected either (if it's an array, the type will be an array)

Reply:
Yeah you're right ^^